sections/futures/Trade/Trade.tsx

```diff
@@ -243,7 +243,10 @@ const Trade: React.FC<TradeProps> = () => {
 				<MarketActionButton onClick={() => setIsDepositMarginModalOpen(true)}>
 					Deposit
 				</MarketActionButton>
-				<MarketActionButton onClick={() => setIsWithdrawMarginModalOpen(true)}>
+				<MarketActionButton
+					disabled={futuresMarketsPosition?.remainingMargin! <= zeroBN}
```

Comment:
Since remainingMargin is of the `Wei` type, I think this should be:
```suggestion
					disabled={futuresMarketsPosition?.remainingMargin?.lte(zeroBN)}
```